src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ CHANGELOG
  * Deprecate the `Symfony\Component\Serializer\Normalizer\ObjectNormalizer` and
    `Symfony\Component\Serializer\Normalizer\PropertyNormalizer` autowiring aliases, type-hint against
    `Symfony\Component\Serializer\Normalizer\NormalizerInterface` or implement `NormalizerAwareInterface` instead
+* Add services and alias deprecation message to `debug:container --name` output
```

Comment:
```suggestion
* Add services and alias deprecation message to `debug:container` output
```

or

```suggestion
* Add services and alias deprecation message to `debug:container [<name>]` output
```


there is no such thing as a `--name` option, but a `name` argument 😄

Reply:
indeed nice catch